tests/keras/backend/backend_test.py

```diff
@@ -1583,8 +1652,10 @@ def test_ctc(self):
         k_inputs = K.variable(inputs, dtype="float32")
         k_input_lens = K.variable(input_lens, dtype="int32")
         k_label_lens = K.variable(label_lens, dtype="int32")
-        res = K.eval(K.ctc_batch_cost(k_labels, k_inputs, k_input_lens, k_label_lens))
-        assert_allclose(res[0, :] if K.backend() == 'theano' else res[:, 0], ref, atol=1e-05)
+        res = K.eval(K.ctc_batch_cost(k_labels, k_inputs, k_input_lens,
+                                      k_label_lens))
+        assert_allclose(res[0, :] if K.backend() == 'theano' else res[:, 0], ref,
```

Comment:
Please make multiple statements when you can rather than breaking very long statements.

This is also the case in other places in this pull request.
```python
if K.backend() == 'theano':
     assert_...
else:
     ....
```